grammar.js

```diff
@@ -47,6 +47,9 @@ noJSX(JavaScript);
  */
 module.exports = grammar(JavaScript, {
   name: "glimmer_javascript",
+
+  externals: ($, previous) => previous.concat([$.raw_text]),
```

Comment:
how does externals work? I should have asked this on the other PR -- but I mostly trust the tests to keep anything from regressing

Reply:
I misunderstood what it meant! I think it means it gets that variable from an external parser. I based this on Vue and Svelte, and I’m pretty sure they have external parsers. Will create PRs to remove it!